build-aux/cades-condo_cpu_only_modules.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Loads all modules that are required to build DCA++ on  ORNL's CADES CONDO cluster
+#
+# Usage: source cades-condo_load_modules.sh
+
+# uncomment if your default module loadout clashes with this
+# module purge
+
+module load env/cades-cnms
+. $SOFTWARECNMS/spack/share/spack/setup-env.sh
+spack load gcc/egooyqw
+module load gcc_6/openmpi/3.1.4
+spack load ninja/gzwd46m
+spack load emacs/qp7x25b
+spack load hdf5/4gmsnjn
+spack load cmake/q76ndqk
+spack load git/kibjjo6
+spack load fftw/kpdartc
+export FFTW_DIR=/software/user_tools/centos-7.2.1511/cades-cnms/spack/opt/spack/linux-centos7-x86_64/gcc-6.5.0/fftw-3.3.8-kpdartcqxfk2kdsbcfdtwin75s24z5uu
+export CXX=mpic++
```

Comment:
Is this necessary? On every other system findMPI should be able to link the mpi library without having to resort to wrappers, which  might to be troublesome if we add hpx wrappers.

Reply:
nb.

- hpx does not require wrappers, it uses standard cmake find_package syntax
- findmpi on many systems actually uses the compiler wrapper to do its thing, though these days you can set MPI_CXX_COMPILER to mpic++ and get the same result most of the time. Since that export is only for a particular machine, it's harmless.